src/utils/components/CloudinitModal/utils/cloudinit-utils.ts

```diff
@@ -129,6 +140,10 @@ export const updateCloudInitRHELSubscription = (
       [ACTIVATION_KEY]: activationKey,
       org: organizationID,
     };
+
+    if (autoUpdateEnabled) {
+      addCloudInitUpdateCMD(draftUserDataObject);
```

Comment:
Why not simply `autoUpdateEnabled && addCloudInitUpdateCMD(draftUserDataObject);` ?

Reply:
okay